src/components/DeleteRoom.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Card, Button, Typography } from '@material-tailwind/react';
+import { fetchRooms, destroyRoom } from '../redux/features/rooms/roomsSlice';
+
+const DeleteRoom = () => {
+  const dispatch = useDispatch();
+  const [rooms, setRooms] = useState([]);
+  // const status = useSelector((state) => state.room.rooms.status);
```

Comment:
- [ ] Kindly, remove this old commented code to improve the repo's cleanliness. 👍